shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -392,6 +392,9 @@ - (UIView*)splashScreenView {
   if (_splashScreenView == nullptr) {
     NSString* launchStoryboardName =
         [[[NSBundle mainBundle] infoDictionary] objectForKey:@"UILaunchStoryboardName"];
+    if (launchStoryboardName == nil) {
+      return nil;
+    }
```

Comment:
Oops.  Sorry about that.

Note that this change won't let you use launch images as the Flutter splash screen. (Explicitly creating a `UIView` with the launch images and using that as the splash screen also would avoid the crash.)